Source/Scene/Model.js

```diff
@@ -418,6 +410,7 @@ define([
          */
         this.silhouetteColor = defaultValue(options.silhouetteColor, Color.RED);
         this._silhouetteColor = new Color();
+        this._silhouetteColorPreviousAlpha = 1.0;
```

Comment:
I'm not sure why this is showing up in the diff since it is in master, can you merge master into your branch?